buildpacks/jvm/src/errors.rs

```diff
@@ -138,5 +112,71 @@ pub(crate) fn on_error_jvm_buildpack(error: OpenJdkBuildpackError) {
             "Could not download OpenJDK distribution.",
             error,
         ),
+        OpenJdkBuildpackError::UnsupportedOpenJdkVersion(artifact_requirement) => {
+            match artifact_requirement.version {
+                HerokuOpenJdkVersionRequirement::Major(major_version) => log_error(
+                    "Unsupported OpenJDK version",
+                    formatdoc! {"
+                        The OpenJDK major version {major_version} you specified in your system.properties file is not supported.
+                        Please specify a supported major version in your system.properties file.
+                    ", major_version = major_version },
```

Comment:
(optional) `formatdoc!` should support interpolation without the explicit second argument of `major_version = major_version` here.

(And same for the entries below)

Reply:
I'll keep this as-is until the error message overhaul - older messages in this file are also using the explicit passing of arguments.